modules/v2x/fusion/configs/ft_config_manager.cc

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/v2x/fusion/configs/ft_config_manager.h"
+
+namespace apollo {
+namespace v2x {
+namespace ft {
+
+FTConfigManager::FTConfigManager() {
+  // std::string file_path = std::string("/configs/perception/perception.conf");
+  // google::SetCommandLineOption("flagfile", file_path_perception.c_str());
+}
```

Comment:
If the constructor is empty, just define it in the header file as `FTConfigManager() = default;`